torch/_inductor/graph.py

```diff
@@ -473,8 +473,7 @@ def __init__(
         # Below field is related to printing debug intermediate tensor values info for debugging
         self.all_codegen_kernel_names: OrderedSet[str] = OrderedSet()
 
-        # state used by wrapper.generate_workspace_allocation()
-        self.allocated_workspaces: Dict[str, Any] = {}
+        # state used by for Kernel.workspace
         self.workspace_id = itertools.count()
```

Comment:
Should this be moved onto WorkspaceArg?

Reply:
It should be local to the graph so codegen is determinstic.